app/http.php

```diff
@@ -773,6 +775,11 @@ function cleanUp(Orchestration $orchestration, Table $activeRuntimes, array $net
             /**
              * Move built code to expected build directory
              */
+            if (empty($localDevice->getFiles($tmpBuildOutput))) {
+                $localDevice->deletePath($tmpLogging . '/logs.txt');
```

Comment:
No need to delete since we throw custom exception

Reply:
Oops, leftover